src/plugin/featureaction/featurestyleaction.js

```diff
@@ -177,7 +190,20 @@ plugin.im.action.feature.StyleAction.prototype.execute = function(items) {
   // notify that the layer needs to be updated
   var layer = os.feature.getLayer(items[0]);
   if (layer) {
-    os.style.notifyStyleChange(layer, items);
+    var source = /** @type {os.source.Vector} */ (layer.getSource());
+    var color = (this.styleConfig['stroke']) ? this.styleConfig['stroke']['color'] : null;
+
+    if (source && color) {
+      source.setColor(items, color); // set the color model's override for these items
+    }
+
+    os.style.notifyStyleChange(
+        layer,
+        items,
+        undefined,
+        undefined,
+        (source && color) // bump the colormodel so dependencies can update/re-render
+    );
```

Comment:
This and the above block look nearly identical. Can we simplify that to a function?

Reply:
The usage of this new colormodel option within "notifyStyleChange()" is pretty embedded in the meat of the function.  I could add a "notifyStyleChangeAndTriggerColorModelListeners()" but because it'd just be calling notifyStyleChange() anyway, it'd be a copy-paste function that'd have to be kept in sync with any future improvements/modifications to notifyStyleChange().

The way it's written now has all of the options to trigger style event(s) on the layer, the source, and/or now the colormodel in a single utility function.  Though often redundant (depending on the interactions of the ways the various plugins set up their listeners), it's pretty nice for getting exactly the event(s) you want.  We'll definitely need to look into how these basic styles and color change events are managed/pushed in the future; but that's asking to refactor all of the listeners in all of the OS plugins... so probably part of the next rewrite.

Reply:
Derp.  Looking at the MR, now I see what you're talking about -- I thought you were referring to the before and after.  Not the Do and Undo in the action itself.  I'll make the updated.